### DISCUSSION TITLE
Native CSS Nesting

### DISCUSSION THREAD

**Discussion by georgetzlp** [stylus/stylus#2811] [Q&A]
How will this project deal with CSS's nesting module?
Will stylus switch to using CSS's native nesting or is it going to keep the current way it does nesting?

In my opinion the nesting that is currently being used is way more useful than CSS's current native nesting.
What do you think about this? Do you prefer Stylus/Sass' nesting or CSS's?

**Comment by vendethiel** [comment]
No point doing the conversion
